04-class-field.tex

```diff
@@ -60,7 +60,7 @@ \section{Property}
 
 \haxe{assets/Property.hx}
 
-For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
+For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
```

Comment:
Are you sure about that colon? I'm pretty sure I use this style (sentences which have listing in between) in several places. I'm also pretty sure I've seen it in other sources.